python/py-tensorflow-datasets/Portfile

```diff
@@ -48,8 +48,7 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-jupyter \
                     port:py${python.version}-mako \
                     port:py${python.version}-pytest \
-                    port:py${python.version}-pytest-xdist \
-                    port:py${python.version}-tensorflow-data-validation
+                    port:py${python.version}-pytest-xdist
 
     test.run        yes
     test.cmd        py.test-${python.branch}
```

Comment:
Steve, since you're updating this port, would you be willing to cleanup the test-related items? (Per Renee's enhancements for the Python portgroup: https://github.com/macports/macports-ports/pull/15001)